examples/federated-learning/players.py

```diff
@@ -97,9 +85,8 @@ def _call_aggregator_fn(self, grads, model):
   def _call_evaluator_fn(self):
     return self.evaluator_fn(self)
 
-  @tfe.local_computation
-  def call_model_fn(self, batches_or_owner):
-    return self.model_fn(batches_or_owner)
+  def call_model_fn(self, batches_or_owner, *args, **kwargs):
```

Comment:
suggest deleting these now, since the abstractmethods will be sufficient

Reply:
including the `_call_evaluator_fn` above & the BaseDataOwner's `call_model_fn`

Reply:
I don't think removed `_call_evaluator_fn` makes sense we're still using pin to owner on that.